share/transformers/edu_ageconsearch.py

```diff
@@ -1,138 +0,0 @@
-import re
```

Comment:
Do we want to remove the old transformer? If we re-transform the old data we will still need this code.